oauthlib/oauth2/rfc6749/endpoints/pre_configured.py

```diff
@@ -159,10 +162,11 @@ def __init__(self, request_validator, token_generator=None,
                                   token_types={'Bearer': bearer})
         RevocationEndpoint.__init__(self, request_validator,
                                     supported_token_types=['access_token'])
+        IntrospectEndpoint.__init__(self, request_validator)
```

Comment:
Is the asymmetry with the `RevocationEndpoint` in `supported_token_types` intentional here?

Reply:
I forgot it, thanks :-)